src/test/java/seedu/task/logic/LogicManagerTest.java

```diff
@@ -113,7 +113,12 @@ private void assertTaskCommandBehavior(String inputCommand, String expectedMessa
         
         //Confirm the ui display elements should contain the right data
         assertEquals(expectedMessage, result.feedbackToUser);
+        
+        for(int i=0;i<model.getFilteredTaskList().size();i++){
+            System.out.print(model.getFilteredTaskList().get(i));
+        }
```

Comment:
Keep in mind this logicmanagertest is no longer in use. Plz follow OOP when testing.